src/components/Button.tsx

```diff
@@ -72,3 +72,31 @@ export const Button = (props: ButtonProps | WaitingButtonProps) => {
 
 export const LinkButton = ({ variant = 'primary', ...props}: LinkButtonBase) =>
   <StyledButton {...props} as='a' variant={variant}>{props.children}</StyledButton>
+
+export const linkColor = '#027EB5';
+export const linkHover = '#0064A0';
```

Comment:
These might be good to throw into the theme file, I have a card to update these at some point (`#026AA1` for the primary, but I don't know the hover.)